test/Microsoft.IdentityModel.Tests/TestSets.cs

```diff
@@ -549,6 +551,12 @@ public static KeyInfoTestSet WrongNamespace
         }
     }
 
+    public class TokenClaimsIdentitiesTestSet
```

Comment:
You can extent SamlTokenTestSet to include IEnumerable<ClaimsIdentity>.
No need for a new class really.